espnet2/spk/pooling/chn_attn_stat_pooling.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import torch.nn as nn
+
+from espnet2.spk.pooling.abs_pooling import AbsPooling
+
+
+class ChnAttnStatPooling(AbsPooling):
+    """
+    Aggregates frame-level features to single utterance-level feature.
+    Proposed in B.Desplanques et al., "ECAPA-TDNN: Emphasized Channel
+    Attention, Propagation and Aggregation in TDNN Based Speaker Verification"
+
+    args:
+        input_size: dimensionality of the input frame-level embeddings.
+            Determined by encoder hyperparameter.
+        output_size: output dimensionality of the pooled utterance-level
+            embedding. In case of this class, output_size is double of input
+            _size becuase it concatenates mean and std.
+    """
+
+    def __init__(
+        self,
+        input_size: int = 1536,
+        output_size: int = 3072,
```

Comment:
Will `output_size` be used somewhere? If it is unused, maybe we can just remove it.

Reply:
oh, I just kept it for `output_size` function. I thought all internal models need this function. 
If this is not the case, it's okay to remove this. 

It's still the input size of following projector, so maybe informative to have it?

Reply:
Do all pooling approaches double the output dimension (e.g., concatenate the mean and variance)? If so, we can just remove the `output_size` as an input argument, and set it as the attribute manually according to the `input_size`.

Reply:
No only a certain ones (normally those that include statistical pooling) double the dimensionality